homeassistant/components/unifi/device_tracker.py

```diff
@@ -51,6 +50,14 @@
     "radio",
     "radio_proto",
     "vlan",
+    "rssi",
+    "signal",
+]
+
+CLIENT_UNRECORDED_ATTRIBUTES = [
+    "ap_mac",
+    "signal",
+    "rssi",
```

Comment:
We don't want these measurements as state attributes. Please move them to separate sensor entities.

Reply:
ap_mac is already a state attribute. from a templating perspective using attributes is much cleaner than separate sensors. Consider the following macro:

{% macro track_device(entity_id) %}
{%- if is_state(entity_id, 'home') -%}
  {%- set ap_mac = state_attr(entity_id, 'ap_mac') -%}
  {%- set signal = state_attr(entity_id, 'signal')|int -%}
  ...
{% endif %}
{% endmacro %}

a single device tracker entity gets passed in, then based on connected ap and signal strength macro can work out what room the device is located in.

was hoping this would be simple enough to add signal and rssi attirbutes to the existing entity, but if not I can live with just patching my local installation.

Reply:
Please close the PR if you don't intend to address the requested changes. Thanks!